interface/VideoEncoderDefs.h

```diff
@@ -152,6 +152,11 @@ typedef struct VideoResolution {
     uint32_t height;
 }VideoResolution;
 
+typedef struct EncMiscParameterHRD {
```

Comment:
Thanks @xuguangxin ,
I just found out VAEncMiscParameterHRD, and this is libva's structure.